app/webpacker/components/CompetitionForm/FormSections/CompetitorLimit.js

```diff
@@ -17,6 +17,7 @@ export default function CompetitorLimit() {
       <ConditionalSection showIf={hasLimit}>
         <InputNumber id="count" min={0} />
         <InputTextArea id="reason" />
+        <InputNumber id="newcomerReservedSpots" min={0} />
```

Comment:
Calculate and display the max number of newcomer reserved spots allowed per the 50% WCRP exception limit